Source/Core/Common/Assert.h

```diff
@@ -36,7 +36,7 @@
   {                                                                                                \
     if (!(_a_))                                                                                    \
     {                                                                                              \
-      if (!PanicYesNo(_fmt_, ##__VA_ARGS__))                                                       \
+      if (!PanicYesNo(_fmt_ "\n\nIgnore and continue?", ##__VA_ARGS__))                            \
```

Comment:
I assume we know that "Ignore and continue?" used to be translated before, but isn't with this change?

Reply:
`_trans` is an empty macro, so does it really make a difference?

Reply:
Oh, I hadn't noticed that... That's probably a problem. What do you think, should we just change it back to the way it was earlier in this PR?

Reply:
It would "_only_" break strings that go through `ASSERT_MSG`, in cases where a translation handler is present (since we're putting "foo" in the translation file and ask for "foo, ignore?" to be translated).
I'm not even sure how to handle this with a translation file where one string is made up from two translations...

Reply:
@CookiePLMonster Yes, it matters. First, without `_trans`, this string won't be picked up by the translations scripts and thus won't get translated by the translators. Second, even if we did mark `\n\nIgnore and continue` as translatable, the string that gets sent to `PanicYesNo` is not that string but that string concatenated to `_fmt_`, which is not a string that exists among the strings that have been translated by translators. I suppose you could get around this by doing `GetStringT("\n\nIgnore and continue?")`, maybe...

Reply:
@BhaaLseN Oh, right, good point. Not only the "Ignore and continue?" part will go untranslated but also the `_fmt_` part. My suggestion in my previous comment would not solve the latter.

Reply:
Maybe it should be left as is for this PR and resolved in the future? Sounds like this PR doesn't regress it, but rather keeps it "equally bad".

Reply:
I'd say put the "Ignore and continue" in `ASSERT` instead of `ASSERT_MSG`, like you originally did in this PR before I reviewed it. It's the only simple solution that doesn't break translations.